lib/src/repository/repository.dart

```diff
@@ -2,7 +2,7 @@ import 'package:gql_exec/gql_exec.dart' as gql;
 import 'package:gql_exec/gql_exec.dart';
 import 'package:gql_link/gql_link.dart';
 
-class BaseRepository {
+class Repository {
```

Comment:
Maybe we can make this an `abstract class Repository` ? Would it make sense to instantiate a Repository class directly?

Reply:
That would be the right move I think. That or making the default constructor private. 

`_Repository() {}`